addons/discuss-button/stick.css

```
@@ -0,0 +1,3 @@
+#navigation {

```

Could yiu re-add `topnav`?
```suggestion
#navigation, #topnav {

```

---

i didnt even noticed it was removed